addons/mass_mailing_sms/models/utm.py

```diff
@@ -31,8 +32,22 @@ def _compute_ab_testing_total_pc(self):
 
     @api.depends('mailing_sms_ids')
     def _compute_mailing_sms_count(self):
+        if self.ids:
+            mailing_sms_data = self.env['mailing.mailing'].read_group(
+                [('campaign_id', 'in', self.ids), ('mailing_type', '=', 'sms')],
+                ['campaign_id', 'ab_testing_enabled'],
+                ['campaign_id', 'ab_testing_enabled'],
+                lazy=False,
+            )
+            ab_testing_mapped_sms_data = {}
+            for data in mailing_sms_data:
+                if data['ab_testing_enabled']:
+                    ab_testing_mapped_sms_data.setdefault(data['campaign_id'][0], []).append(data['__count'])
+        else:
+            ab_testing_mapped_sms_data = dict()
         for campaign in self:
             campaign.mailing_sms_count = len(campaign.mailing_sms_ids)
```

Comment:
It could probably use counters from read_group instead of re-using the 2Many field, like what is done in mailing.